tasks/utils.py

```diff
@@ -352,7 +352,8 @@ def get_version(
 ):
     version = ""
     pipeline_id = os.getenv("CI_PIPELINE_ID")
-    if pipeline_id and pipeline_id.isdigit():
+    project_name = os.getenv("CI_PROJECT_NAME")
+    if pipeline_id and pipeline_id.isdigit() and project_name == "datadog-agent":
```

Comment:
There are some constants on the top of this file; one of them is `REPO_PATH = f"{ORG_PATH}/datadog-agent"` - would it make sense to extract the `datadog-agent` string into a new constant and use it here as well as in other constants/places throughout this file?

Reply:
Uhm i'm not exactly you what do you suggest ? ``REPO_PATH`` is the local file path of the datadog-agent repository but ``CI_PROJECT_NAME`` is set by the gitlab runner. Meaning that on datadog-agent and datadog-agent-buildimages ``REPO_PATH`` are the same value but ``CI_PROJECT_NAME`` aren't